### PR TITLE
(DOP-2600) Create new video banner and group banner components into folder

### PR DESCRIPTION
**Pull request by gjchong25**
### Stories/Links:

[DOP-2600](https://jira.mongodb.org/browse/DOP-2600)

### Staging Links:

[Atlas Staging Link](https://docs-mongodb-org-stg.s3.us-east-2.amazonaws.com/master/cloud-docs/grace.chong/DOP-2600/getting-started/index.html)

### CODE REVIEW

File: src/components/Banner/CTABanner.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+              <Icon glyph="Play" fill="#007CAD" />
+            </div>
+          }
+        >
+          {children.map((child, i) => (
+            <ComponentFactory {...rest} key={i} nodeData={child} />
+          ))}
+        </StyledBanner>
+      </a>
+    </>
```
Comment (rayangler) [inline src/components/Banner/CTABanner.js:61]: We can probably remove the fragment here and make the `a` tag the outermost element.

File: src/components/Banner/CTABanner.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const CTABanner = ({ nodeData: { children, options }, ...rest }) => {
+  return (
+    <>
+      <a href={options.url}>
+        <StyledBanner
+          className={cx(videoBannerStyling)}
+          variant={LeafyVariant.Info}
+          image={
+            <div css={playIconStyling}>
+              <Icon glyph="Play" fill="#007CAD" />
```
Comment (rayangler) [inline src/components/Banner/CTABanner.js:52]: (Nit) The fill here can be `uiColors.blue.base`

**Review by rayangler — COMMENTED** [review]
Just a couple of nitpicks from me but otherwise looks great! And props for re-organizing

**Review by allisonmui9 — APPROVED** [review]
LGTM! Small comment that when it loads the elements of the component flicker out of alignment.

**Review by allisonmui9 — APPROVED** [review]
looks good. thanks grace!

**Review by rayangler — APPROVED** [review]
LGTM!

**Pull request merged**